tests/test_user_flow.py

```diff
@@ -3,9 +3,19 @@
 Test user flow
 """
 
+import pytest
 from bs4 import BeautifulSoup
 from flask import url_for
-import pytest
+from mock import Mock, patch
+
+
+mock_openid_config = Mock()
+mock_openid_config.return_value = {
+    'authorization_endpoint':
+    'http://dex.example.com:5556/auth',
+    'discovery_url':
+    'http://dex.example.com:5556',
+    'client_id': 'testid'}
```

Comment:
If these values don't fit on the same line as the keys, please indent them one level